### ISSUE TITLE
Unable to render

### ISSUE THREAD

**Issue by kamilturek** [Ars-Linguistica/mlconjug3#404]
**Describe the bug**
`mlconjug3` is unable to render the answer in Italian. Other languages work fine.

**To Reproduce**
```
$ mlconjug3 -l it andare
2023-02-11 23:46:52,696   ERROR         cli.py  144: An error occurred: unable to render int; a string or other renderable object is required
Conjugations not displayed. Please check the input verbs and language.
```

**Expected behavior**
Display the answer.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - macOS 13.1

**Comment by maroph** [comment]
Same is true for me on Debian 12.1 using Python 3.11.2
All versions since 3.9.0 show this behavior. I can't check older version with Python 3.11

It still works for me with the following workaround:

```
import json
import mlconjug3

conjugator = mlconjug3.Conjugator(language='it')
verb = conjugator.conjugate('essere')
print(json.dumps(verb.conjug_info, ensure_ascii=False, indent=2))
```

**Comment by SekouDiaoNlp** [comment]
Hi @kamilturek and @maroph.

Thanks for letting me know about the issue.

I will investigate this week and update the software as soon as possible.

Cheers.

**Comment by SekouDiaoNlp** [comment]
Hi @maroph and @kamilturek .

I have just released mlconjug3 version 3.11.0 which fixes the rendering issue with Italian verbs. Thank you very much for reporting the issue.

Cheers.
